settings:
  units: 
    orientation: 'degrees' # euler_degrees, euler_radians, quaternion

  controlled_frame: 'tool0'

  # Path tolerance (set to null for default)
  path_tolerance: null
    #position_error: [0.001, 0.001 ,0.001] #[x,y,z]
    #orientation_error: [0.01, 0.01 ,0.01] #[x,y,z]
    #twist_error:
    #  linear:  [0.001, 0.001 ,0.001] #[x,y,z]
    #  angular:  [0.01, 0.01 ,0.01] #[x,y,z]
    #acceleration_error:
    #  linear:  [0.001, 0.001 ,0.001] #[x,y,z]
    #  angular:  [0.01, 0.01 ,0.01] #[x,y,z]

  # Goal tolerance (set to null for default)
  goal_tolerance: null
    #position_error: [0.001, 0.001 ,0.001] #[x,y,z]
    #orientation_error: [0.01, 0.01 ,0.01] #[x,y,z]
    #twist_error:
    #  linear:  [0.001, 0.001 ,0.001] #[x,y,z]
    #  angular:  [0.01, 0.01 ,0.01] #[x,y,z]
    #acceleration_error:
    #  linear:  [0.001, 0.001 ,0.001] #[x,y,z]
    #  angular:  [0.01, 0.01 ,0.01] #[x,y,z]

  # Time tolerance (set to null for default)
  goal_time_tolerance: null

trajectory:
  - time: 0
    positions: [-70.0, -115.0, -88.0, -65.0, 88.0, 90.0]
    velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    #accelerations:
    #efforts:

  - time: 2.0
    positions: [-70.0, -105.0, -88.0, -65.0, 88.0, 90.0]
    velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    #accelerations:
    #efforts:

  - time: 3.0
    positions: [-90.0, -105.0, -88.0, -65.0, 88.0, 90.0]
    velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    #accelerations:
    #efforts:
  
  - time: 5.0
    positions: [-90.0, -115.0, -88.0, -65.0, 88.0, 90.0]
    velocities: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    #accelerations:
    #efforts: